sidebars.js

```diff
@@ -148,6 +148,7 @@ module.exports = {
             "components/modeler/web-modeler/launch-web-modeler",
             "components/modeler/web-modeler/model-your-first-diagram",
             "components/modeler/web-modeler/context-pad",
+            "components/modeler/web-modeler/git-sync",
```

Comment:
Is this the preferred placement/developer journey by the Modeler team?

Reply:
That was the one that we agreed on.